Mathlib/Geometry/Manifold/VectorBundle/MDifferentiable.lean

```diff
@@ -61,6 +61,97 @@ theorem mdifferentiableAt_totalSpace (f : M → TotalSpace F E) {x₀ : M} :
         (fun x ↦ (trivializationAt F E (f x₀).proj (f x)).2) x₀ := by
   simpa [← mdifferentiableWithinAt_univ] using mdifferentiableWithinAt_totalSpace _ f
 
+section coordChange
+
+variable [(x : B) → AddCommMonoid (E x)] [(x : B) → Module 𝕜 (E x)]
+variable (e e' : Trivialization F (π F E)) [MemTrivializationAtlas e] [MemTrivializationAtlas e']
+  [VectorBundle 𝕜 F E] [ContMDiffVectorBundle n F E IB] (hn : 1 ≤ n)
```

Comment:
Wouldn't it make more sense to assume `[ContMDiffVectorBundle 1 F E IB]` here? Otherwise, `n` is not relevant, and there is no way it can be guessed by Lean.